app/views/items/show.html.erb

```diff
@@ -6,22 +6,25 @@
 <h4><%= @headline %></h4>
 <ul id="review_list">
   <% if @reviews.empty? %>
```

Comment:
Making the change.  This was this way because we wanted the bullet point to say "no reviews yet", as opposed to a <p> tag.